lib/cog/commands/sort.ex

```diff
@@ -1,76 +1,81 @@
 defmodule Cog.Commands.Sort do
-  use Cog.Command.GenCommand.Base, bundle: Cog.embedded_bundle, execution: :once
+  use Cog.Command.GenCommand.Base, bundle: Cog.embedded_bundle
+  alias Cog.Command.Service.MemoryClient
 
   @moduledoc """
-  Sorts the given inputs.
-
-  ## Example
-
-      @bot #{Cog.embedded_bundle}:sort 3 2 1 5 4
-      > [1, 2, 3, 4, 5]
-      @bot #{Cog.embedded_bundle}:sort --asc 4.5 1.8 0.032 0.6 1.5 0.4
-      > [0.032, 0.4, 0.6, 1.5, 1.8, 4.5]
-      @bot #{Cog.embedded_bundle}:sort --desc Life is 10 percent what happens to us and 90% how we react to it
-      > [what, we, us, to, to, react, percent, it, is, how, happens, and, Life, %, 90, 10]
-      @bot #{Cog.embedded_bundle}:rules --for-command=rules| sort --field=rule
-      > {
-          "rule": "when command is operable:permissions with option[user] == /.*/ must have operable:manage_users",
-          "id": "12345678-abcd-efgh-ijkl-0987654321ab",
-          "command": "operable:permissions"
-        }
-        {
-          "rule": "when command is operable:permissions with option[role] == /.*/ must have operable:manage_roles",
-          "id": "87654321-mnop-qrst-uvwx-0123456789ab",
-          "command": "operable:permissions"
-        }
-        {
-          "rule": "when command is operable:permissions with option[group] == /.*/ must have operable:manage_groups",
-          "id": "24680135-azby-cxdw-evfu-ab0123456789",
-          "command": "operable:permissions"
-        }
+  Sorts the given list of input in ascending order by default.
+
+  ## Usage
+
+    sort [options] [fields...]
+
+    Fields are used to pick which values to sort by. If two keys have the same
+    value the values of the next key are compared and so on. If no fields are
+    provided, items are intellegently sorted based on their contents.
+
+  ## Options
+
+    --asc, -a    sort in ascending order
+    --desc, -d   sort in descending order
+
+  ## Examples
+    @cog seed '[{"a": 1}, {"a": 3}, {"a": 2}]' | sort
+    > [{"a": 1}, {"a": 2}, {"a": 3}]
+
+    @cog seed '[{"a": 1}, {"a": 3}, {"a": 2}]' | sort --desc
+    > [{"a": 3}, {"a": 2}, {"a": 1}]
+
+    @cog seed '[{"a": 3, "b": 4}, {"a": 1, "b": 4}, {"a": 2, "b": 6}]' | sort b a
+    > [{"a": 1, "b": 4}, {"a: 3, "b": 4}, {"a": 2, "b": 6}]
```

Comment:
I would like to talk about standardizing our usage info. We're kinda all over the board with it. I dropped a message in general to get some input.